### ISSUE TITLE
How to contribute to DrupalGap core

### ISSUE THREAD

**Issue by luxio** [signalpoint/DrupalGap#823]
Hi,

should a pull request to DrupalGap core include also `drupalgap.js`and `drupalgap.min.js`?

Which tool do you use to minify `drupalgap.js`?

**Comment by signalpoint** [comment]
@luxio Yes, it's helpful if Pull Requests include the .js and .min.js binaries. I have script which helps with that:

```
cd mobile-application
./scripts/update_binaries
```

If everything else has been committed/pushed/pulled down so your local 7.x-1.x branch is clean, the above script can be run to quickly commit/push both binaries.

`npm` is used to compile the `src/` directory into the binaries:

```
cd mobile-application
npm install
grunt
```

You only need to run the `npm install` once, then all subsequent times just run `grunt` and that'll listen for changes to the `src/` directory, and then compile it down into the binaries.
